drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -446,6 +454,10 @@ msgid "Reset"
 msgstr ""
 
 #: public/js/drag_and_drop.js
+msgid "Show Answer"
+msgstr ""
+
+#: public/js/drag_and_drop.js
```

Comment:
It's great to see you updated the translation file for `en`. Could you please update  `eo` file as well? See [this](https://github.com/edx/i18n-tools/blob/master/i18n/dummy.py#L103) for one possible method of generating dummy translations.